docker/scripts/p4app.py

```diff
@@ -0,0 +1,89 @@
+from mininet.net import Mininet
+from mininet.topo import Topo, SingleSwitchTopo
+
+from p4_mininet import P4Host, P4Switch
+from p4_program import P4Program
+
+from p4runtime_switch import P4RuntimeSwitch
```

Comment:
any reason why `P4RuntimeSwitch` is not in the same module as `P4Host` and `P4Switch`?